extensions/sql/contract-definition/schema/src/main/java/org/eclipse/dataspaceconnector/sql/contractdefinition/schema/SqlContractDefinitionTables.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (c) 2022 Daimler TSS GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Daimler TSS GmbH - Initial API and Implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.sql.contractdefinition.schema;
+
+public final class SqlContractDefinitionTables {
```

Comment:
Minor nit: typically we're using interfaces to `store` constants, you're already doing it for config keys :)